### PR TITLE
Add exclamation in description in README.md

### PR DESCRIPTION
**Pull request by phillipj**
~Assuming this will get closed as well due to missing label and stale not being configured to consider said label.~

### CODE REVIEW

**Comment by phillipj** [comment]
_Update:_ initially thought this would get closed because it didn't have the `stalled` label and issues has been configured to be closed after 1 day.

Turns out the Action classifies issues as "stale" only if they've got the label configured via [`stale-pr-label`](https://github.com/phillipj/stale-actions-without-label-config/blob/master/.github/workflows/stale.yml#L14) attached. All other issues are ignored.

In other words, no need to use the [`labels-only` option](https://github.com/actions/stale/blob/master/action.yml?rgh-link-date=2020-06-28T19%3A33%3A58Z#L30) which seems required at first glance.
